src/physical-and-chemical-data/data/data.ts

```diff
@@ -0,0 +1,58 @@
+import { PhysicalProperties } from '../interfaces/physical-properties';
```

Comment:
You can rename this file to `inorganic-compounds.ts`.

Rest looks good.

Reply:
Okay sir